src/extensions/magic_mount.rs

```diff
@@ -0,0 +1,53 @@
+mod chunk;
+
+type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
+
+impl crate::application::CoLink {
+    pub(crate) async fn _mm_create_entry(
+        &self,
+        key_name: &str,
+        payload: &[u8],
+    ) -> Result<String, Error> {
+        let split_key = key_name.split('$').collect::<Vec<&str>>();
+        let token = split_key[split_key.len() - 1];
+        let key_name = split_key[0];
```

Comment:
I think here we should assume all things before **the last** `:$` are the key name